.github/workflows/test.yml

```diff
@@ -172,16 +172,17 @@ jobs:
       - uses: actions/checkout@v5
         with:
           fetch-tags: true
-      - uses: actions/setup-python@v6
+      - name: Install uv & Python
+        uses: astral-sh/setup-uv@v6
         with:
-          python-version: '3.12'
-      - name: Install dependencies
-        run: python3 -m pip install --upgrade build twine
+          python-version: "3.13"
+      - name: Ensure uv.lock is up to date
+        run: env --unset=UV_FROZEN uv lock --check
       - name: Build
         run: |
-          python3 -m build .
-          python3 -m build ext/
+          uv build .
+          uv build ext/
       - name: Check package metadata
         run: |
-          twine check --strict dist/*
-          twine check --strict ext/dist/*
+          uvx twine check --strict dist/*
+          uvx twine check --strict ext/dist/*
```

Comment:
Can we get rid of twine in favour of `uv publish`?

Reply:
Publishing (`release.yml`) doesn't use twine, it uses `pypa/gh-action-pypi-publish@v1.13.0`.

This is just doing some checks on every PR build. [According to twine docs](https://twine.readthedocs.io/en/latest/#twine-check): "Checks whether your distribution’s long description will render correctly on PyPI"